src/hawkbit-client.c

```diff
@@ -741,19 +742,23 @@ gboolean install_complete_cb(gpointer ptr)
  * feedback and call software_ready_cb() callback on success.
  *
  * @param[in] data Artifact* to process
- * @return NULL is always returned
+ * @return if download thread waited for installation to finish, TRUE if installation and download
+ *         succeeded, FALSE otherwise. If download thread did not wait for installation to finish,
+ *         TRUE if download succeeded, FALSE otherwise.
```

Comment:
``` suggestion 
 * @return pointer to gboolean being TRUE if download succeeded, FALSE otherwise.
 *         Note that if the download thread waited for installation to finish ('run_once' mode),
 *         TRUE means both installation and download succeeded.
```

Reply:
Thanks, applied.